webdriver/tests/execute_script/execute.py

```diff
@@ -70,3 +87,26 @@ def test_abort_by_user_prompt_twice(session, dialog_type):
     assert session.alert.text == "Bye"
 
     session.alert.accept()
+
+
+def test_execute_pinned(session):
+    pin_script(session, "One", "return 1;")
+    response = execute_script(session, name="One")
+    assert_success(response, 1)
+
+
+def test_name_and_script(session):
+    pin_script(session, "One", "return 1;")
+    response = execute_script(session, "return 1;", name="One")
+    assert_error(response, "invalid argument")
+
+
+def test_no_such_pinned_script(session):
+    response = execute_script(session, name="One")
+    assert_error(response, "no such script")
+
```

Comment:
What happens if you call a script and pass in a name of a pinned script but that method doesn't exist?

Reply:
I'm not following. Which method?